src/shared.ts

```diff
@@ -85,6 +85,15 @@ export const defaultProblem: IProblem = {
     tags: [] as string[],
 };
 
+export class Solution {
```

Comment:
Seems only `src/leetCodeSolutionProvider.ts` referenced the Solution, put it into the `leetCodeSolutionProvider`